### ISSUE TITLE
OpenAL Soft Extensions

### ISSUE THREAD

**Issue by Squareys** [NuclearMonster/openal-website#2]
Hello @rcgordon, Hello all!

I recently looked through the [openal extension registry](http://icculus.org/alextreg/) and found that extensions by [OpenAL soft](https://github.com/kcat/openal-soft) are not listed.

As of what I can tell, the following extensions are defined and implemented by OpenAL soft: 

```
    AL_SOFT_block_alignment, AL_SOFT_buffer_samples, AL_SOFT_buffer_sub_data,
    AL_SOFT_deferred_updates, AL_SOFT_direct_channels, AL_SOFT_loop_points,
    AL_SOFT_MSADPCM, AL_SOFT_source_latency, AL_SOFT_source_length,
    ALC_SOFTX_device_clock, ALC_SOFTX_HRTF, ALC_SOFT_loopback,
    ALC_SOFTX_midi_interface, ALC_SOFT_pause_device
```

Is there a reason why they are not listed, or is there a chance that someone will do that?
Some of them seem pretty well documented: http://kcat.strangesoft.net/openal-extensions/

Thanks in advance!
Sincerely, Squareys.

**Comment by TimeDoctor** [comment]
This is not a bug for this website.

**Comment by Squareys** [comment]
Alright, sorry about that. I hoped you know the guys mantaining the extension registry, there doesn't seem to be a way to contact them. Since @rcgordon registered one or the other extension, I thought, he might know something?

**Comment by NuclearMonster** [comment]
We may integrate the registry and revisit it then.

**Comment by icculus** [comment]
Yeah, we're figuring out how to migrate this to openal.org. Stay tuned.  :)

**Comment by Artoria2e5** [comment]
Hi uh @icculus, the registry seems to have ceased working with no results returned. A look in the browser network tab shows HTTP 500 Internal Server Error.
* * *
A recent alsoft release links to @raulshc's https://github.com/Raulshc/OpenAL-EXT-Repository. It's a nice resource, but you don't get Ryan's semantic search tools, only GitHub search.

**Comment by Raulshc** [comment]
Hello all,

please, consider integrate the entire extension database from https://github.com/Raulshc/OpenAL-EXT-Repository into the official web. In this way everything will be available through the official site.

Thank you
